Sources/SwiftInspectorCommands/Tests/TypealiasCommandSpec.swift

```diff
@@ -0,0 +1,85 @@
+// Created by Francisco Diaz on 3/25/20.
+//
+// Copyright (c) 2020 Francisco Diaz
+//
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Foundation
+import Nimble
+import Quick
+@testable import SwiftInspectorCore
+
+final class TypealiasCommandSpec: QuickSpec {
+  override func spec() {
+    var fileURL: URL!
+    var path: String!
+
+    beforeEach {
+      fileURL = try? Temporary.makeFile(content: "final class Some: SomeType { }")
+      path = fileURL?.path ?? ""
+    }
+
+    afterEach {
+      try? Temporary.removeItem(at: fileURL)
+    }
+
+    describe("run") {
+      context("with no arguments") {
+        it("fails") {
+          let result = try? TestTask.run(withArguments: ["typealias"])
```

Comment:
I'm a bit nervous about how this actually works (see https://github.com/fdiaz/SwiftInspector/pull/7#issuecomment-604760576). It seems like it would fail under these circumstances, since the binary wouldn't exist.

1. Clean
2. Only build `SwiftInspectorCommands`
3. Run tests for `SwiftInspectorCommands`

But we can dig into this separately.

Reply:
Answered there 😉